keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/config.h

```diff
@@ -0,0 +1,96 @@
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* #define DEBUG_LED_PIN      GP25 */
+#define CRC8_USE_TABLE
+#define CRC8_OPTIMIZE_SPEED
+
+/* Reset. */
+#define RESET_PIN GP22
+#define RESET_PIN_DOUBLE_TAP_RESET_TIMEOUT 1000U
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP25
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+/* split things */
+#define SPLIT_POINTING_ENABLE
+#define SPLIT_HAND_PIN      GP28  // high = left, low = right
+
+/* Key matrix configuration. */
+#define MATRIX_ROW_PINS \
+    { GP4, GP5, GP6, GP7, GP8, GP9 }
+#define MATRIX_COL_PINS \
+    { GP10, GP11, GP12, GP13, GP14, GP15 }
+
+/* WS2812 RGB LED strip input and number of LEDs */
+#define WS2812_PIO_USE_PIO1
+#define WS2812_DI_PIN       GP21
+#define RGBLED_NUM          62
+#define RGB_MATRIX_LED_COUNT 62
+#define RGBLIGHT_SPLIT
+#define RGB_MATRIX_SPLIT \
+    { 32, 30 }
+#define RGBLED_SPLIT \
+    { 32, 30 }
+
+/* serial.c configuration for split keyboard */
+#define SERIAL_PIO_USE_PIO0
+#define SERIAL_USART_FULL_DUPLEX  // Enable full duplex operation mode.
+#define SERIAL_USART_TX_PIN      GP0
+#define SERIAL_USART_RX_PIN      GP1
+#define SERIAL_USART_TIMEOUT     100  // USART driver timeout. default 100
+#define SERIAL_USART_SPEED       921600
+#define SERIAL_USART_PIN_SWAP
+
+/* i2c config for oleds */
+#define I2C_DRIVER        I2CD1
+#define I2C1_SDA_PIN      GP2
+#define I2C1_SCL_PIN      GP3
+#define I2C1_CLOCK_SPEED  400000
+#define I2C1_DUTY_CYCLE FAST_DUTY_CYCLE_16_9
+
+/* SPI config */
+#define SPI_DIVER                            SPID0
+#define SPI_SCK_PIN                          GP18
+#define SPI_MOSI_PIN                         GP19
+#define SPI_MISO_PIN                         GP16
+
+/* pmw3360 config */
+#define POINTING_DEVICE_CS_PIN               GP17
+
+/* encoder pins */
+#define ENCODERS_PAD_A { GP27 }
+#define ENCODERS_PAD_B { GP26 }
+
+/*****************************************************/
+/*  P M W 3 3 6 0   t r a c k b a l l   s e n s o r  */
+/*****************************************************/
+#define PMW33XX_LIFTOFF_DISTANCE 0x08
+#define POINTING_DEVICE_ROTATION_180
+#undef ROTATIONAL_TRANSFORM_ANGLE
+#define ROTATIONAL_TRANSFORM_ANGLE -15
+#define CHARYBDIS_MINIMUM_DEFAULT_DPI 1600
+#define CHARYBDIS_DRAGSCROLL_DPI 200
+#define CHARYBDIS_DRAGSCROLL_REVERSE_Y
+
+/*********************************/
+/*  E n c o d e r   c o n f i g  */
+/*********************************/
+#define ENCODER_RESOLUTION 1
```

Comment:
These are settings that should be set at keymap level.  

Worst case, in a post config file like the oled brightness